src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1556,10 +1523,10 @@ k
 
         $.each(airport.terrain || [], (elevation, terrainLevel) => {
             max_elevation = Math.max(max_elevation, elevation);
-            const color = `rgba(${prop.ui.terrain.colors[elevation]}, `;
+            const color = `rgba(${prop.ui.terrain.COLOR[elevation]}, `;
```

Comment:
It's not my preference either, but it can be moved to the appropriate place at a later date.